cmd/kubelet/app/options/globalflags_linux.go

```diff
@@ -73,6 +73,8 @@ func addCadvisorFlags(fs *pflag.FlagSet) {
 	registerDeprecated(global, local, "storage_driver_table", deprecated)
 	registerDeprecated(global, local, "storage_driver_secure", deprecated)
 	registerDeprecated(global, local, "storage_driver_buffer_duration", deprecated)
+	registerDeprecated(global, local, "containerd-namespace", deprecated)
+	registerDeprecated(global, local, "disable_root_cgroup_stats", deprecated)
```

Comment:
/hold

I'm not sure surfacing **more** cadvisor flags in the kubelet flagset is a goal. @dashpole, does this match your expectations?

cc @mtaufen

Reply:
We probably don't want to expose `disable_root_cgroup_stats` in the kubelet, as it would break parts of it.

I also don't see a reason why `containerd-namespace` would be set as IIRC the cri-containerd shim always puts kubernetes-owned pods in a specific namespace.  Can you expand on why you want to set the flag?

Reply:
In alibaba  ,we used [pouch](https://github.com/alibaba/pouch) as the container runtime, in cadvisor the container which created by pouch only can handle by containerd. Because of the legacy, pouch used  'default ' as the namespce, but the containerd used 'k8s.io' as the namespace, so we need this’containerd-namespace' flag in containerd.
About the disable_root_cgroup_stats , we don't need it now, i just add all the flags which defined in cadvisor to flagset, if don't want to expose this in the kubelet, i can delete it in this pr.

Reply:
Ok.  @liggitt I think it is reasonable to add the containerd-namespace flag.  Adding more flags from cAdvisor to the kubelet is certainly not a goal, but given that there still isn't a concrete plan to replace cAdvisor, I we should permit it.

Reply:
is it not reasonable to require the k8s.io namespace be used?

Reply:
@dashpole DO i need delete the 'disable_root_cgroup_stats' in this pr  or not ?

Reply:
> DO i need delete the 'disable_root_cgroup_stats' in this pr or not ?

I would expect so, given the "We probably don't want to expose disable_root_cgroup_stats in the kubelet, as it would break parts of it." comment

Reply:
I don't think we should include disable_root_cgroup_stats.  Multiple parts of the kubelet would not function correctly with that set.

Reply:
@dashpole @liggitt Now, only add containerd-namespace, PTAL